.eslintrc.json

```diff
@@ -6,15 +6,22 @@
     "airbnb-base/legacy"
   ],
   "globals": {
+    "afterEach": true,
+    "beforeEach": true,
+    "afterAll": true,
+    "beforeAll": true,
     "describe": true,
+    "expect": true,
     "it": true,
     "fit": true,
-    "expect": true,
-    "xit": true,
+    "test": true,
     "xdescribe": true,
-    "fdescribe": true
-  }, 
+    "fdescribe": true,
+    "xit": true,
+    "xtest": true
+  },
   "rules": {
+    "linebreak-style": "off",
```

Comment:
`eol` should be set to `\n`. My bad for not setting it into the `.vscode/settings.json` or an `.editorConfig` file. I'll do it so this won't be an issue.